docs/docs/widgets/star-rating.md

```diff
@@ -65,12 +65,12 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 <div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
+| Show on desktop | Makes the component visible in desktop view. | You can set it with the toggle button or dynamically configure the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`. |
```

Comment:
Here you need to update the content of the expected value column, it should be:

"You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression." 

instead of:

"You can set it with the toggle button or dynamically configure the value by clicking on **fx** to set the value `{{true}}` or `{{false}}`."

PS: Here I used the quotes to highlight the sentence, you don't need to use them in the docs. :)